### DISCUSSION TITLE
Special orb size

### DISCUSSION THREAD

**Discussion by jankovic016** [runelite/runelite#15808] [Ideas]
I would appreciate option to set special orb or any information orb  larger. I often missclick it and then taking unnecessary damage, because i ¨missclick on minimap or turn off my run. 

Hope devs would be OK with this.

Thanks.

**Comment by gonX** [comment]
You could use the core RuneLite Plugin "Stretched Mode" to accomplish this, by setting the plugins "Resizable scaling" setting to a value greater than 0. It will resize all of your UI elements, however.
